templates/CRM/Contact/Form/Merge.tpl

```diff
@@ -273,6 +271,25 @@
     CRM.$( "#main_" + blockname + "_" + blockId + "_overwrite" ).html( label );
   }
 
+  // Find a matching main contact location block by entity, location and type
```

Comment:
Suppose we should set a good example. I can't see any examples of a full docblock for a JS function. But I'll put one in here.